### PR TITLE
Add Remote Domain and sentances, responses and tests

### PR DESCRIPTION
**Pull request by tombo12354**

### CODE REVIEW

File: sentences/en/_common.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ expansion_rules:
   area: "[the] {area}"
   what_is: "(what's | whats | what is)"
   brightness: "{brightness} [percent]"
-  turn: "(turn | switch | change)"
+  turn: "(turn | switch | change | shut)"
```
Comment (tetele) [inline sentences/en/_common.yaml:51]: "shut" is only used for shutting something off, not on. Maybe it's better to add it in the HassTurnOff intent (e.g. `(<turn> | shut) off ...`).

Comment (tombo12354) [reply sentences/en/_common.yaml:51]: That's a good point, I agree that makes more sense.

Comment (tombo12354) [reply sentences/en/_common.yaml]: Updated

**Comment by tombo12354** [comment]
Not sure I understand what's failing on the tests. Reading through it, it seems to be failing in that it cannot find remote files for other languages. Is there somewhere else to update?

**Comment by tetele** [comment]
@tombo12354 would you care to update this PR or should we close it? I'm not sure remotes are used only for TVs, though.

**Pull request closed without merging**